hugo/content/playground/utils.ts

```diff
@@ -15,27 +15,6 @@ export async function share(grammar: string, content: string): Promise<void> {
   await navigator.clipboard.writeText(url.toString());
 }
 
-export function throttle<T>(milliseconds: number, action: (input: T) => void) {
```

Comment:
Throttling is a way to minimize the rebuilds of the grammar. I used it to avoid consuming too much resources. Otherwise it would be after each keystroke triggered document build.

Reply:
We still have this, it's been unrolled from the function to a couple of cases where `setTimeout` and `clearTimeout` are invoked directly (in common.ts). The current delays are 200ms as well, otherwise yeah we would have way too many requests.

Reply:
Great!

Reply:
I think while I was hacking away at things I pulled this out, but it might be nice to bring it back, and use the single function to replace the two usages, since it's effectively the same.

Reply:
@Lotes I went ahead and brought back a version of `throttle` to manage the timeouts. Internally I keep information on previous timeouts, so there's callbacks to manage the state there anymore. Works nicely 🙂 .